tests/hypothesis_strategies/alternative_creation/test_searchspace.py

```diff
@@ -97,3 +101,34 @@ def test_searchspace_creation_from_dataframe(df, parameters, expected):
     else:
         with pytest.raises(expected):
             SearchSpace.from_dataframe(df, parameters)
+
+
+@given(
+    parameters=st.lists(
+        numerical_discrete_parameter(min_value=0.1, max_value=1.0),
```

Comment:
Why not allowing min_value=0.0?

Reply:
Mostly just to limit the test case a bit. You see, if you put 0.0 instead, then the simplex can contain an arbitrary number of parameters, because each parameter can just contribute 0.0 to the sum (provided is has 0.0 as a value). But putting 0.1, we know that the simplex can span at most 10 parameters, due to the total being 1.0. The alternative – which would work just as well – is to limit the number of parameter in the list created by hypothesis. I'd be fine with both solutions. Which one do you prefer? Or do you have an even better idea?

Reply:
Ok, I get the point. I'd prefer the variant where we explicitly limit the number of parameters, as the case of a parameter having a value of 0 seems like something we should test.
Did you actually test of what you are worried about actually happens/makes any difference?

Reply:
agree with alex, limits via min value seems like a detour to achieve whats possible also by setting the corresponding entry in the draw AND having 0.0 in there seems important to me as this will happen for lots of use cases

Reply:
Ok, valid point. Changed it accordingly 👍🏼 Now restricting the number of parameters instead